.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Description
+<!--
+- What this pull request does.
+- Bug fix, new feature, documentation change, etc.
+-->
+
+### Checklist
+- [ ] Corresponding issue has been opened
+- [ ] New tests added
+- [ ] Feature flags / configuration added
+
+### Other Related Issues
+<!--
+- The primary issue this PR addresses should be part of the PR title.
+- If there are other tickets related to this PR, reference them here with context of how they are relevant.
+-->
+None
+
+### Verification steps
+<!--
+Describe how to validate your changes.
+- Include screen shots if applicable.
+- Note if migrations are required.
+-->
+
+### AFTER YOU MERGE
+
+1. Cut a release tag using the GitHub workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
```

Comment:
> notify in the #aselo-deploys Slack channel.
This is a new step we did not follow before and seems redundant. (I might have missed the reasons, if any, from the discussion in retro).

Reply:
I assume Steve means that the Github workflow will post a notification, rather than the engineer posting a notification, but I agree that the wording is ambiguous here.

Reply:
Updated